cmd/arc_summary/arc_summary3

```diff
@@ -221,6 +223,32 @@ elif sys.platform.startswith('linux'):
 
         return descs
 
+def handle_unraisableException(exc_type, exc_value=None, exc_traceback=None,
+                               err_msg=None, object=None):
+   handle_Exception(exc_type[0], exc_type[1], exc_traceback)
+
+def handle_Exception(ex_cls, ex, tb):
+    if ex is IOError:
+        if ex.errno == errno.EPIPE:
+            sys.exit()
+    if ex is KeyboardInterrupt:
+        sys.exit()
+
+    if ex_cls is BrokenPipeError:
+      devnull = os.open(os.devnull, os.O_WRONLY)
+      os.dup2(devnull,sys.stdout.fileno())
+      os.dup2(devnull,sys.stderr.fileno())
+      # It turns out that while sys.exit() triggers an exception
+      # not handled message on Python 3.8+, os._exit() does not.
+      os._exit(0)
+    raise ex
+
+# It turns out that for Python 3.8+, they reworked how some systems work,
+# such that just using sys.excepthook doesn't solve the problem any more.
+if hasattr(sys,'unraisablehook'): # Python 3.8+
```

Comment:
missing space after comma, also the EOL-comment seems a bit superfluous

Reply:
Thanks!

Initially, I added the comment because it was pretty opaque to me (and several people I consulted) what was going on with the initial behavior of apparently ignoring sys.excepthook for a non-unraisable exception, but on reading it again, I agree that it doesn't add much value, so I'll drop it.